.travis.yml

```diff
@@ -1,16 +1,21 @@
-sudo: false
+sudo: required # due to manual install of make 4.1
```

Comment:
I was gonna dismiss this but I now see that Travis finally supports sudo even for containerized environments. Previously requiring sudo would force your build to be run outside a container which routinely delayed it for up to 30min (until a slot became available). All good now.

Reply:
Maybe at some point it would be worthwhile to suss out the performance of running our own docker instance there since they also support that. That way we wouldn't be as beholden to their rather old build environments. Not that I'm volunteering, I don't know much about docker, and no real bandwidth to change that... :)